packages/ui/components/ui/identity-key-component.tsx

```diff
@@ -0,0 +1,18 @@
+import { IdentityKey } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/keys/v1/keys_pb';
+import { CopyToClipboardIconButton } from './copy-to-clipboard-icon-button';
+import { bech32IdentityKey } from '@penumbra-zone/types';
+
+/**
+ * Renders an `IdentityKey` for a validator, along with a copy button.
+ */
```

Comment:
Can you list an example one so we know what an identity key looks like?

Reply:
Does [this](https://github.com/penumbra-zone/web/pull/620/commits/442519949ead1c1d286f5b1fff39fcd033b1caa1) work? We don't usually construct identity keys ourselves — we just grab them off of `Validator` objects.